driver/driver_test.go

```diff
@@ -767,7 +787,11 @@ var _ = Describe("On requests from docker daemon", func() {
 
 		BeforeEach(func() {
 			_, dockerNetID, containerID = setupNetworksAndEndpoints(contrailController, docker)
-			dockerNet, err := docker.NetworkInspect(context.Background(), dockerNetID, false)
+			inspectOptions := dockerTypes.NetworkInspectOptions{
+				Scope:	 "",
+				Verbose: false,
+			}
+			dockerNet, err := docker.NetworkInspect(context.Background(), dockerNetID, inspectOptions)
```

Comment:
Since this got much more verbose, I think we should wrap this whole 5-line block in some helper function